docs/source/daml-script/index.rst

```diff
@@ -0,0 +1,191 @@
+.. Copyright (c) 2019 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Script
+###########
+
+**WARNING:** DAML Script is an experimental feature that is actively
+being designed and is *subject to breaking changes*.
+We welcome feedback about DAML script on
+`our issue tracker <https://github.com/digital-asset/daml/issues/new?milestone=DAML+Script>`_
+or `on Slack <https://hub.daml.com/slack/>`_.
+
+DAML scenarios provide a simple API for experimenting with DAML models
+and getting quick feedback in DAML studio. However, scenarios are run
+in a special process and do not interact with an actual ledger. This
+means that you cannot use scenarios to test other ledger clients,
+e.g., your UI or :doc:`DAML triggers </triggers/index>`.
+
+DAML script addresses this problem by providing you with an API with
+the simplicity of DAML scenarios and all the benefits such as being
+able to reuse your DAML types and logic while running against an
+actual ledger. This means that you can use it to test automation
+logic, your UI but also for ledger initialization where scenarios
```

Comment:
```suggestion
logic and your UI. DAML Script can also be used for ledger initialization on all ledger implementations.
```
It took me a while to understand the sentence. I don't think it's really necessary to mention here that scenarios can be used to initialize the sandbox. IMO, it is more likely to cause confusion rather than really help the documentation of DAML Script.